.github/workflows/SCHEDULE-buildDrupalDocker.yml

```
@@ -93,8 +94,8 @@ jobs:
           labels: 'drupal.version.label="${{ matrix.drupal-version }}",php.version.label="${{ matrix.php-version }}", postgresql.version.label="${{ matrix.pgsql-version }}"'
       ## Build the image tagged as latest which is the highest version combo that we feel is well supported.
       - uses: mr-smithers-excellent/docker-build-push@v6
-        name: Build latest using 10.3.x-dev, PHP 8.3, PgSQL 16
-        if: ${{ matrix.drupal-version == '10.3.x-dev' && matrix.php-version == '8.3' && matrix.pgsql-version == '16' }}
+        name: Build latest using 10.3.x-dev, PHP 8.3, PgSQL 17
+        if: ${{ matrix.drupal-version == '10.3.x-dev' && matrix.php-version == '8.3' && matrix.pgsql-version == '17' }}
```

Bump the Drupal version for latest in the schedules docker drupal base image as well.
```suggestion
        name: Build latest using 11.1.x-dev, PHP 8.3, PgSQL 17
        if: ${{ matrix.drupal-version == '11.1.x-dev' && matrix.php-version == '8.3' && matrix.pgsql-version == '17' }}
```